optimizely/optimizely.py

```diff
@@ -1357,13 +1357,27 @@ def _identify_user(self, user_id: str) -> None:
             self.logger.error(enums.Errors.INVALID_OPTIMIZELY.format('identify_user'))
             return
 
+        if not self.odp_manager.enabled:
+            self.logger.debug('ODP identify event is not dispatched (ODP disabled).')
+            return
```

Comment:
I believe this is checked in odp_manager. Do we need to check here again?

Reply:
This allows `create_user_context` to remain non-blocking if odp is disabled. Seems like a worthwhile optimization?

Reply:
@andrewleap-optimizely I see. Probably not in server sdks. They'll be blocked anyway in following decide call even if avoid it here :)